index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+    <title>Climatechange-2022</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <div class="left-nav">
+                <a><img class="mobile-logo" src="images/Union.png" alt=""></a>
+                <div class="logo-div">
+                    <a href="#homepage"><img class="desktop-logo" src="images/my-icon.png" alt=""></a>
+                    <div class="logo-name"> <a href="#homepage">CC Global <br>Campaigns</a></div>
+                </div>
+            </div>
+            <div class="mid-nav">
+                <a class="nav-link" href="#about2">About</a>
+                <a class="nav-link" href="http://">Program</a>
+                <a class="nav-link" href="http://">Join</a>
+                <a class="nav-link" href="http://">Sponsor</a>
+                <a class="nav-link" href="http://">News</a>
+            </div >
+            <button class="right-nav">CC Campaign</button>
+           
+        </nav>
+    </header>
+    <section id="homepage">
+        <div><h2>"Trees for better lives"</h2></div>
+        <div><h1 class="Theme-header">Climate Change Global Conference-2022</h1></div>
```

Comment:
- [ ] For making  The HTML markup is well structured and follows a clear hierarchy,I recommend you to change `Hx` tag orders Use `h1` tag for main title and `h2` for subtitle.